params/governance_params.go

```diff
@@ -47,10 +46,6 @@ var (
 
 	DefaultMintedKLAY = big.NewInt(0).Mul(big.NewInt(defaultMintedKLAYInSton), big.NewInt(Ston))
```

Comment:
The above 4 variables are not used also.

Reply:
@aidan-kwon Can I add a commit to remove those four and them merge to dev?

Reply:
Cleared in https://github.com/klaytn/klaytn/pull/1356.